appserver/admingui/payara-theme/src/main/resources/branding/masthead.inc

```diff
@@ -94,26 +112,4 @@
             />
         </sun:panelGroup>
     </facet>
-    <!facet userInfoExtras>
```

Comment:
Looks like you removed line 97, which is perhaps an attempt at an opening tag? (I'm not familiar with woodstock) but not removed the corresponding close tag

Reply:
nevermind it was the split view in github confusing matters